### PR TITLE
MGDDOC-66: Test cases for concepts and RN in RHOAM 

### PR DESCRIPTION
**Pull request by joaedwar**
# Description

Test cases for the following Jiras:
* [MGDDOC-66](https://issues.redhat.com/browse/MGDDOC-66)
* [MGDDOC-64](https://issues.redhat.com/browse/MGDDOC-64)
* [MGDDOC-19](https://issues.redhat.com/browse/MGDDOC-19)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by joaedwar** [comment]
Hi @tonyxrmdavidson @trdoyle81. This failed but I am not sure why. Would you please look it over and let me know where I went wrong?

**Comment by joaedwar** [comment]
@psturc Please review

File: test-cases/tests/documentation/i16-verify-partial-ag-rhmi-config.md [hunk 1]
```diff
@@ -1,32 +0,0 @@
----
```
Comment (psturc) [inline test-cases/tests/documentation/i16-verify-partial-ag-rhmi-config.md:1]: Hey @joaedwar if this test case no longer valid for RHMI?

If it's still valid, I would keep that in the repo.

File: test-cases/tests/documentation/i01-verify-release-notes.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
```
Comment (psturc) [inline test-cases/tests/documentation/i01-verify-release-notes.md:1]: I would rather avoid deleting RHMI test cases, if we could use them in a future.

So if you think this test case might be still valid for RHMI in a future, you can split this test case to 2 similar test case files (1 for RHMI, 1 for RHOAM):

```
# For RHMI
test-cases/tests/documentation/i01a-verify-release-notes.md
# For RHOAM
test-cases/tests/documentation/i01b-verify-release-notes.md
```

Also note that the **title** needs to be updated in those test cases
For RHMI:
**# I01A - Verify release notes**
for RHOAM:
**# I01B - Verify release notes**

File: test-cases/tests/documentation/i16-verify-partial-ag-rhmi-config.md (hunk 1 above)
Comment (joaedwar) [reply test-cases/tests/documentation/i16-verify-partial-ag-rhmi-config.md:1]: We are no longer updating RHMI docs, just maintaining them. This is for RHOAM docs. So, I think this should be deleted.

File: test-cases/tests/documentation/i01-verify-release-notes.md (hunk 2 above)
Comment (joaedwar) [reply test-cases/tests/documentation/i01-verify-release-notes.md:1]: I don't think we are creating release notes for RHMI anymore.

**Pull request merged**